microcosm_eventsource/resources.py

```diff
@@ -38,4 +37,16 @@ class SearchEventSchema(PageSchema):
     max_clock = fields.Integer()
     parent_id = fields.UUID()
     sort_by_clock = fields.Boolean()
+    sort_clock_in_ascending_order = fields.Boolean()
```

Comment:
At this point, I'd consider having a three-state parameter that can be None/Ascending/Descending instead of having two separate parameters and a really long name.